gc/base/Configuration.cpp

```diff
@@ -305,6 +305,9 @@ MM_Configuration::initializeRunTimeObjectAlignmentAndCRShift(MM_EnvironmentBase*
 		}
 #endif /* defined(S390) */
 
+		/* When shouldForceToLowMemoryHeapCeilingShiftIfPossible is turned on, we will set the shift value to 3 except when it's greater than 3 */
+		if ((canChangeShift) && extensions->shouldForceToLowMemoryHeapCeilingShiftIfPossible && shift < DEFAULT_LOW_MEMORY_HEAP_CEILING_SHIFT)
+			shift = DEFAULT_LOW_MEMORY_HEAP_CEILING_SHIFT;
```

Comment:
please reformat to emphasize action forced by `shouldForceToLowMemoryHeapCeilingShiftIfPossible` (plus multiple coding standards fixes):
```
		/* When shouldForceToLowMemoryHeapCeilingShiftIfPossible is turned on, we will set the shift value to 3 except when it's greater */
                if (extensions->shouldForceToLowMemoryHeapCeilingShiftIfPossible) {
                    if (canChangeShift && (shift < DEFAULT_LOW_MEMORY_HEAP_CEILING_SHIFT)) {
                        shift = DEFAULT_LOW_MEMORY_HEAP_CEILING_SHIFT;
                    }
                }
```

Reply:
Also https://github.com/eclipse/openj9/pull/10305 made a decision always set `shouldForceToLowMemoryHeapCeilingShiftIfPossible` by default (if no related options specified) if runs in container. 

@youngar @rwy0717  Should this decision be made on OMR level?

Reply:
I have moved the decision to here. At the very least it's better than having that decision made in the gc parser.